### DISCUSSION TITLE
Question- What am I missing?? Trying to scrape and print off the name and ages on all three wikis.

### DISCUSSION THREAD

**Discussion by virtualstorm24** [community/community#64312] [Programming Help]
### Select Topic Area

Question

urls = ["https://en.wikipedia.org/wiki/List_of_victims_of_the_September_11_attacks_(A%E2%80%93G)_",
"https://en.wikipedia.org/wiki/List_of_victims_of_the_September_11_attacks_(H%E2%80%93N)_",
"https://en.wikipedia.org/wiki/List_of_victims_of_the_September_11_attacks_(O%E2%80%93Z)_"]

for url in urls:
    response = requests.get(url)
    soup = bs(response.content, 'html.parser')

    table = soup.select(".wikitable.sortable")

    if table:
        rows = table[0].find_all('tr')
        for row in rows[1:]:
            cells = row.find_all('td')
            if len(cells) == 2:
                name = cells[0].get_text(strip=True)
                age = cells[1].text(strip=True)
                print("Name:", name)
                print("Age:", age)
    else:
        print("No table found", url)

**Comment by vacisdev** [comment]
You just needed to correct your method from the `cell`, instead of `text` you needed to write `get_text`.

```tables = soup.select(".wikitable.sortable")

if table:
    table = tables[0]
    rows = table[0].find_all('tr')
    for row in rows[1:]:
        cells = row.find_all('td')
        if len(cells) == 2:
            name = cells[0].get_text(strip=True)
            age = cells[1].get_text(strip=True)
            print("Name:", name)
            print("Age:", age)
else:
    print("No table found", url)```

**Comment by ALISHERpy** [comment]
maybe you must to write get_text.
i have never use "cell" for it

**Reply by Dasturchi-Yigit** [reply]
yeah,i on you
